coreneuron/io/nrn_setup.cpp

```diff
@@ -180,6 +188,10 @@ std::vector<NetCon*> netcon_in_presyn_order_;
 /// Only for setup vector of netcon source gids
 std::vector<int*> netcon_srcgid;
 
+/// If a netcon_srcgid is negative, need to determine the thread when
+/// in order to use the correct neg_gid2out[tid] map
+std::vector<std::vector<int> > netcon_negsrcgid_tid;
```

Comment:
I'd rename this to `std::vector<std::vector<int>> netcon_negsrcgid_tids;`, it's a bit ambiguous further down the line wrt  `std::vector<int> netcon_negsrcgid_tid; // entries only for negative srcgids` + callback

Reply:
I agree the name is not the best. Neither is ```netcon_srcgid```. A verbose version of ```netcon_srcgid``` is  ```nrnthread_netcon_srcgid``` as the size of the vector is nthread and each element is a an ```n_netcon`` length vector of int (which should be changed to use an std vector of vectors. Similarly I could accept ```nrnthread_netcon_negsrcgid_tid```.
Let me know if you think this is worthwhile for this pull request.

Reply:
`nrnthread_netcon_negsrcgid_tid` sounds good to me, also fine to leave it as it is.